.ci/Jenkinsfile

```
@@ -545,41 +563,42 @@ def runTestingCommand(def args = [:]) {
 }
 
 def buildWorkspaceContext(def args = [:]) {
-  return "${args.testingType}-${args.linuxDistro}-PHP-${args.phpVersion}-${args.componentTestsAppHostKind}"
+  return "${args.testingType}-${args.linuxPackageType}-PHP-${args.phpVersion}-${args.componentTestsAppHostKind}"
   + (args.shouldUseSignedBinaries ? "-signed-binaries" : "")
 }
 
 def buildLabel(def args = [:]) {
   return "${args.testingType} testing for"
-  + " Linux: ${args.linuxDistro}"
+  + " Linux: ${args.linuxPackageType}"
   + ", PHP: ${args.phpVersion}"
   + ", componentTestsAppHostKind: ${args.componentTestsAppHostKind}"
+  + ", testGroup: ${args.testGroup}"
```

I could not find the reference for testGroup but `componentTestsGroup`:

```suggestion
  + ", componentTestsGroup: ${args. componentTestsGroup}"
```

---

Good catch - fixed.